.gitignore

```diff
@@ -10,3 +10,7 @@
 /smoketest*.conf
 /smoketeststorage
 /storage
+CMakeFiles/
+CMakeCache.txt
+Makefile
+*.cmake
```

Comment:
For consistency, please prefix all of these with `/` and sort them above alphabetically.

Also, if *.cmake is just for cmake_install.cmake, please just expand that out here. If there are other possible *.cmake files, then disregard this.